src/main/java/net/minecraft/server/EULA.java

```diff
@@ -70,7 +70,7 @@ private void c() {
                     Properties properties = new Properties();
 
                     properties.setProperty("eula", "false");
```

Comment:
Prob should make this `true`

Reply:
No, that's against Minecraft EULA I think.